files/zh-cn/web/css/_colon_target/index.html

```diff
@@ -73,11 +73,11 @@ <h4 id="结果">结果</h4>
 
 <p>{{EmbedLiveSample('一个内容列表', 500, 300)}}</p>
 
-<h3 id="纯CSS高亮">纯CSS高亮</h3>
+<h3 id="纯CSS高亮">纯 CSS 高亮</h3>
```

Comment:
此处步修改会影响 170 行的 EmbedLiveSample，所以请考虑更新所有带有中文的 id（当更新 h 标签时），可通过额外的脚本完成。

```suggestion
<h3 id="纯_css_高亮">纯 CSS 高亮</h3>
```